[JS] Others : Data Types in JavaScript (#1600)
girlscript/winter-of-contributing: JavaScript/Topics/5.Syntax/1.Data_Types/README.md

    age:50, 
    eyeColor:"blue"};
```
## 2. Arrays

> Arrays are collection of values of similar data type.

Reviewer: In javascript, arrays can be a collection of different data types

Author: Sir, I've made the changes now review it.

> Arrays are collection of values of similar data type. In JavaScript, array can also be a collection of data of different data types.

Reviewer: ```suggestion
Arrays are collection of values of similar or different data types.
```

JavaScript Arrays are written with square bracket<kbd>[ ]</kbd>, seperated by commas.

__Example:__
```javascript
const bike = ["BMW","Honda","Suzuki"];
```

> Array indexes are zero-based, means the first item is at [0].

Reviewer: ```suggestion
Array indexes are zero-based, means the first item is at  arrayName[0].
```
